pulsar-client-tools/src/main/java/org/apache/pulsar/shell/PulsarShell.java

```diff
@@ -132,32 +147,61 @@ public PulsarShell(String args[], Properties props) throws IOException {
             exit(1);
             throw new IllegalArgumentException(e);
         }
+
+        pulsarShellDir = computePulsarShellFile();
+        Files.createDirectories(pulsarShellDir.toPath());
```

Comment:
we should probably write something to the output to tell that we created a directory that did not exist